plugins/portal-counts.js

```diff
@@ -30,6 +30,28 @@ window.plugin.portalcounts = {
   CENTER_Y: 100,
 };
 
+// Determine if a point is inside of a polygon.
+// ray-casting algorithm based on
+// https://observablehq.com/@tmcw/understanding-point-in-polygon
+window.plugin.portalcounts.portalinpolygon = function(portal,LatLngsObjectsArray) {
```

Comment:
@modos189 

1. Shouldn't we include some pip algorithm in IITC api? It is used in many plugins already. 
2. I'm going to open separate issue to discuss another opportunity: what if we provide general portal-filtering API?
For example, it'd be possible to use more convenient region selection tools, like these: #27

Reply:
It would be great if common functions were included in API. We can replace this code when corresponding API appears